### ISSUE TITLE
Dice scripting for character sheets (aka Macros)

### ISSUE THREAD

**Issue by Flamma** [fariapp/fari-app#420]
## 🚩 Is your feature request related to a problem? Please describe

I want to contribute with WoD character sheets for the platform. Starting with Vampire, I want to replicate this sheet:

![image](https://user-images.githubusercontent.com/1144708/181303930-7cdd862e-c0ca-4fc8-8754-41514fd231ee.png)
(only the top par is pasted).

The basic system is easy: you take one attribute rating, you add it to the ability rating, and you throw that number of dices, counting successes.

This is what I have done so far:

![image](https://user-images.githubusercontent.com/1144708/181304569-d0b8c87c-355e-4499-ab2f-f8cb0ead8759.png)

I used `slot tracker block` just because is visually more similar, but I'm aware it's not its purpose. Instead, a `dice pool block` seems closer to what I want to do, but it's still not good, because it would roll just that pool, but I need to roll two pools together. We could press on both pools and read two lines of dice, but that would be more confusing to my players.

For being able to implement all RPG systems, I think Fari needs a more flexible way of defining rolls, that could be customized for every game. Like macros on most VTT systems.

**Comment by RPDeshaies** [comment]
You can actually combine two pools together by "right clicking" on the dice pool button and then clicking "Roll" at the bottom left corner when you are ready!

Tell me if that fixes your issue

**Comment by Flamma** [comment]
I'm so sorry, I just discovered the way to add two dice pools, solving the main problem.

Still, I think maybe a redesign of rolls and the ability of scripting would be positive.

Using pools makes the sheet not very readable, and having to select the pools with right click, then close the sheet to be able to roll what is selected is not very intuitive.

**Comment by RPDeshaies** [comment]
I agree, I'm currently working on a redeisgn as we speak, so stay tuned for this!
